modules/conversion-center/classes/render/core-render.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+namespace Elementor\Modules\ConversionCenter\Classes\Render;
+
+use Elementor\Utils;
+
+/**
+ * Class Core_Render.
+ *
+ * This class handles the rendering of the Link In Bio widget for the core version.
+ *
+ * @since 3.23.0
+ */
+class Core_Render extends Render_Base {
+
+	public function render( \Elementor\Modules\ConversionCenter\Widgets\Link_In_Bio $widget ): void {
+		$settings = $widget->get_settings_for_display();
+
+		/**
+		 * Map args to internals
+		 */
+
+		//  Identity Image
+		$identity_image_props_style = $settings['identity_image_style'] ?? 'profile';
+		$identity_image_value       = $settings['identity_image'] ?? [];
+
+		//  Bio Heading
+		$bio_heading_output    = '';
+		$bio_heading_props_tag = $settings['bio_heading_tag'] ?? 'h2';
+		$bio_heading_value     = $settings['bio_heading'] ?? '';
+
+		//  Bio Title
+		$bio_title_output    = '';
+		$bio_title_props_tag = $settings['bio_title_tag'] ?? 'h2';
+		$bio_title_value     = $settings['bio_title'] ?? '';
+
+		//  Bio Description
+		$bio_description_output = '';
+		$bio_description_value  = $settings['bio_description'] ?? '';
+
+		// Icons
+		$icons_value = $settings['icon'] ?? [];
+
+		// CTAs
+		$ctas_value = $settings['cta_link'] ?? [];
+
+		/**
+		 * Checks
+		 */
+
+		$has_bio_description = ! empty( $bio_description_value );
+		$has_bio_heading     = ! empty( $bio_heading_value );
+		$has_bio_title       = ! empty( $bio_title_value );
+		$has_ctas            = ! empty( $ctas_value );
+		$has_icons           = ! empty( $icons_value );
+		$has_identity_image  = ! empty( $identity_image_value ) && ( ! empty( $identity_image_value['url'] || ! empty( $identity_image_value['id'] ) ) );
+
+		// Render conditions
+		if ( ! $has_bio_heading ) {
+			return;
+		}
+		?>
+		<div class="e-link-in-bio__content-container">
+			<?php if ( $has_identity_image ) :
+				$widget->add_render_attribute( 'identity', [
+					'class' => [ 'e-link-in-bio__identity', "e-link-in-bio__identity--{$identity_image_props_style}" ],
+				] );
+				?>
+				<figure <?php echo esc_attr( $widget->get_render_attribute_string( 'identity' ) ); ?>>
+					<?php if ( ! empty( $identity_image_value['id'] ) ) {
+						echo wp_get_attachment_image( $identity_image_value['id'], 'medium', false, [
+							'class' => 'e-link-in-bio__identity-image',
+						] );
+					} else {
+						$widget->add_render_attribute( 'identity_image', [
+							'alt'   => '',
+							'class' => 'e-link-in-bio__identity-image',
+							'src'   => esc_url( $identity_image_value['url'] ),
+						] );
+						?>
+						<img <?php echo esc_attr( $widget->get_render_attribute_string( 'icon-link' ) ); ?> />
```

Comment:
Where are the render attributes for `icon-link` created?

Reply:
@willhowat can you fix this?